scripts/docker/install-python.bash

```diff
@@ -39,7 +39,7 @@ impyla==0.16.0 \
 pyodps==0.8.3 \
 jupyter==1.0.0 \
 notebook==6.0.0 \
-sqlflow==0.7.0 \
+sqlflow==0.9.0 \
```

Comment:
Is this sqlflow pip package actually built from pysqlflow repo?

Reply:
Yes, push a new tag of pysqlflow by [make release](https://github.com/sql-machine-learning/pysqlflow#release), so that PyPI would build a pip package with the new tag.